src/components/FinishLevel/Modal.jsx

```diff
@@ -90,73 +77,17 @@ const PopUp = ({
                         />
                       </div>
                       <p className="text-lg">90 Level1 Tokens</p>
-                    </div>
+                    </div> */}
                   </div>
                   <br />
                   <p className="text-base">
-                    Let&#39;s mint your NFT and add your Level1 tokens to your
-                    wallet.
+                    Enter your wallet address to mint your completion NFT.
                   </p>
                 </div>
               </div>
               <div className="space-y-2 mt-5 sm:mt-6">
-                <div className="alert alert-warning">
-                  <div className="flex-1">
-                    <svg
-                      xmlns="http://www.w3.org/2000/svg"
-                      fill="none"
-                      viewBox="0 0 24 24"
-                      className="w-6 h-6 mx-2 stroke-current"
-                    >
-                      <path
-                        strokeLinecap="round"
-                        strokeLinejoin="round"
-                        strokeWidth="2"
-                        d="M12 9v2m0 4h.01m-6.938 4h13.856c1.54 0 2.502-1.667 1.732-3L13.732 4c-.77-1.333-2.694-1.333-3.464 0L3.34 16c-.77 1.333.192 3 1.732 3z"
-                      ></path>
-                    </svg>
-                    <label>
-                      Make sure your wallet is connected to the Rinkeby Test
-                      Network for demo.
-                    </label>
-                  </div>
-                </div>
-                <button
-                  type="button"
-                  className={
-                    `btn btn-primary btn-outline btn-block` +
-                    (addingTokens
-                      ? " loading"
-                      : addTokensComplete
-                      ? " opacity-50 "
-                      : "")
-                  }
-                  onClick={() => awardTokens()}
-                  disabled={addTokensComplete}
-                >
-                  {addTokensComplete ? "Tokens Added" : "Add Tokens"}
-                  {addTokensComplete && <CheckIcon className="h-6 w-6 ml-2" />}
-                </button>
-                <button
-                  type="button"
-                  className={
-                    `btn btn-accent btn-outline btn-block` +
-                    (minting ? " loading" : mintComplete ? " opacity-50 " : "")
-                  }
-                  onClick={() => mintNft()}
-                  disabled={mintComplete}
-                >
-                  {mintComplete ? "Minted NFT" : "Mint NFT"}
-                  {mintComplete && <CheckIcon className="h-6 w-6 ml-2" />}
-                </button>
+                <MintNFT metaData="https://gateway.pinata.cloud/ipfs/QmZuD7NRnz9ms3MYoEUMxC6gLyjBDcwcQSVx6tYn8oC1sE" />
```

Comment:
Can we move this url into environment variable?

Reply:
This will be updated to pull from the DB. It will be different for each learning journey.